src/components/button/Button.jsx

```diff
@@ -0,0 +1,5 @@
+import "./Button.scss";
+
+export default function Button({text, type}) {

```

Comment:
To avoid setting `type` all of the time in your component, like:
```jsx
<Button type="primary" text="Learn More" />
```

You can create a default params to the Button component, that the default value will be `primary`, like so:

```suggestion
export default function Button({ text, type = "primary" }) {
```

and you can now call it in the most simple way:
```jsx
<Button text="Learn More" />
```

Reply:
prettier is activated but it is not putting in the white spaces